packages/standalone-analyzer/src/cli.ts

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
```

Comment:
This file looks like it's uncovered by tests, but it's not - there is `cli.test.ts` which tests against the transpiled version of `cli.ts`